MAUI/DataForm/migration.md

```diff
@@ -0,0 +1,1085 @@
+<td>LabelText</td>
+<td>Gets or sets the editor’s label text.</td>
+</tr>
+
+<tr>
+<td>{{'[PlaceHolderText](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormItem.html#Syncfusion_XForms_DataForm_DataFormItem_PlaceHolderText)'| markdownify }}</td>
+<td>PlaceholderText</td>
+<td>Gets or sets the prompt string displayed or the water mark to provide hint for users in the editors.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>PlaceholderColor</td>
+<td>Gets or sets the color of prompt string displayed or the water mark to provide hint for users in the editors.</td>
+</tr>
+
+<tr>
+<td>{{'[ShowLabel](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormItem.html#Syncfusion_XForms_DataForm_DataFormItem_ShowLabel)'| markdownify }}</td>
+<td>ShowLabel</td>
+<td>Gets or sets a value indicating whether to show the label to the editor.</td>
+</tr>
+
+<tr>
+<td>{{'[IsReadOnly](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormItem.html#Syncfusion_XForms_DataForm_DataFormItem_IsReadOnly)'| markdownify }}</td>
+<td>IsReadOnly</td>
+<td>Gets or sets a value indicating whether editor can be enabled or disabled.</td>
+</tr>
+
+<tr>
+<td>{{'[EditorFontSize](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormItem.html#Syncfusion_XForms_DataForm_DataFormItem_EditorFontSize)'| markdownify }}</td>
+<td>EditorTextStyle</td>
+<td>Gets or sets the style of editor text, that used to customize the text color, font, font size, font family and font attributes.</td>
+</tr>
+
+<tr>
+<td>{{'[ErrorMessageColor](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormItem.html#Syncfusion_XForms_DataForm_DataFormItem_ErrorMessageColor)'| markdownify }}</td>
+<td>ErrorLabelTextStyle</td>
+<td>Gets or sets the style of error message label , that used to customize the text color, font, font size, font family and font attributes.</td>
+</tr>
+
+<tr>
+<td>{{'[ValidationLabelStyle](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormItem.html#Syncfusion_XForms_DataForm_DataFormItem_ValidationLabelStyle)'| markdownify }}</td>
+<td>ValidMessageLabelTextStyle</td>
+<td>Gets or sets the style of valid message label, that used to customize the text color, font, font size, font family and font attributes.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>LabelTextStyle</td>
+<td>Gets or sets the style of editor label text, that used to customize the text color, font, font size, font family and font attributes.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>Background</td>
+<td>Gets or sets the background to the data form item layout which is used to render label and editor for the data field.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>DefaultLayoutSettings</td>
+<td>Gets or sets the value of layout (holds label and editor) settings which is used to customize the label position, label width and editor width.</td>
```

Comment:
<td>Gets or sets the value of the layout (holds label and editor) settings used to customize the label position, label width, and editor width.</td>